python/pyspark/ml/tests.py

```diff
@@ -616,6 +622,7 @@ def test_save_load(self):
         tvsModel.save(tvsModelPath)
         loadedModel = TrainValidationSplitModel.load(tvsModelPath)
         self.assertEqual(loadedModel.bestModel.uid, tvsModel.bestModel.uid)
+        self.assertEqual(len(loadedModel.validationMetrics), len(tvsModel.validationMetrics))
```

Comment:
It would be better to compare each `validationMetric` in the list not only the length of `validationMetrics`   cc @jkbradley